### ISSUE TITLE
Allow multiple string placeholder for the "Quick  app" feature

### ISSUE THREAD

**Issue by xchs** [leokhoa/laragon#61]
I am trying to execute a command chain using _Quick app_ and need the string placeholder `%s` **several times** there, e.g.

`laragon\usr\sites.conf`:
```console
Foobar=composer create-project example/foobar %s 1.2.* && curl https://example.com/baz.php -o %s/bar/baz.php
```
However, when I run this _Quick app_, Laragon shows me the following error message and crashes:

![laragon1](https://user-images.githubusercontent.com/1218766/52707693-98600c80-2f88-11e9-87b0-d17bf0f92592.png)

If I remove one string placeholder from the command chain it works as expected, e.g.

```console
Foobar=composer create-project example/foobar foo 1.2.* && curl https://example.com/baz.php -o %s/bar/baz.php
```

Is there a way that it can work with multiple placeholders as well?

**Comment by leokhoa** [comment]
@xchs : I've fixed it in Laragon 4.0.10

**Comment by xchs** [comment]
@leokhoa Thanks a lot! 👍
